test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of fiware-cygnus (FI-WARE project).
+#
+# fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+# fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+# http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact:
+# iot_support at tid.es
+#
+__author__ = 'Iván Arias León (ivan.ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+Feature: Stored in ckan new notifications per column from context broker using matching table with patterns
+  As a cygnus user
+  I want to be able to store in ckan new notifications per column from context broker using matching table with patterns
+  so that they become more functional and useful
+
+  @happy_path  @matching_table
+  Scenario Outline: stored new notifications in ckan with different matching_table patterns
+    Given copy properties.json file from "epg_properties.json" to test "ckan-sink" and sudo local "false"
+    And configuration of cygnus instances with different ports "true", agents files quantity "1", id "test" and in "column" mode
+    And copy flume-env.sh, matching table file from "matching_table.conf", log4j.properties, krb5.conf and restart cygnus service. This execution is only once "false"
+    And verify if cygnus is installed correctly
+    And "ckan" is installed correctly
+    And create a new organization "org_patterns" with a dataset "<new_dataset>"
+    And create a new resource "<new_destination>" with "1" attributes called "temperature", attribute type "celcius", attribute data type "json" and metadata data type "json"
+    And update real values in resource "<resource>" and service path "<service_path>" to notification request
+    When receives a notification with attributes value "random", metadata value "False" and content "<content>"
+    Then I receive an "OK" http code
+    And changes new destination "<new_destination>" where to verify in dataset "<new_dataset>"
+    And Verify that the attribute value is stored in ckan
+    And Verify the metadatas are stored in ckan
+    And delete matching table file
+  Examples:
+    | service_path | resource        | new_destination | new_dataset     | content |
```

Comment:
new_dataset --> new_service_path as stated in PR https://github.com/telefonicaid/fiware-cygnus/pull/409